docs/testplan/session-management.md

```diff
@@ -0,0 +1,31 @@
+|                   | Authorization 2        | Second, if there were no issues with authorization from the previous step and if the session is valid from the new table given the session ID in the cookie, then the User struct should be received from the database using the UserID in the session |          |   
+|                   | Logout- Delete Session | If logout endpoint is called, then session should be deleted from database                                                                                                                                                                             |          |   
+|                   | Cookie Deletion        | Cookie should be deleted only after session is deleted from database                                                                                                                                                                                   |          |   
+|                   | Inactivity Timout      | Session should be invalidated if user is inactive for a set period of time                                                                                                                                                                             |          |   
+|                   | Closing the Browser    | A session should be invalidated once the user closes their browser for a session                                                                                                                                                                       |          |   
+|                   | New IP Address         | If a session is created from a new IP(different than previous session), then there should be two-factor authentication to verify the user                                                                                                              |          |   
+|                   | Valid Authorization    | If the session is valid from the new table given the session ID in the cookie, then User struct should be received from the database using the UserID in the session                                                                                   |          |   
+| Multiple Sessions | Logout- Delete Session | If a user is logged into multiple sessions, then all sessions should be invalidated once the user logs out from one of the sessions                                                                                                                    |          |   
+|                   | Reset Password         | If a user resets their password, then all sessions should be invalidated once the user successfully resets their password                                                                                                                              |          |   
```

Comment:
Reset password is the only one that should invalidate all sessions. Logout, inativity timer and closing the browser should only invalidate the current but not all sessions.